imagebuildah/executor.go

```diff
@@ -406,13 +406,12 @@ func (b *Executor) Build(ctx context.Context, stages imagebuilder.Stages) (image
 							// arg expansion, so if the previous stage
 							// was named using argument values, we might
 							// not record the right value here.
-							rootfs := flag[7:]
+							rootfs := strings.TrimPrefix(flag, "--from=")
```

Comment:
OK, maybe I've weekend brain still but don't the old and new lines return the same thing?  I'm not sure how this fixes the issue?

Reply:
You're not wrong - that's just a change that I think makes the line easier to follow.

Reply:
I concur that it's better code and I'd like to keep it, but other than test changes/additions, I see no other code changes?  Are you missing a change that actually fixes the issue?

Reply:
The removal of a `break` in a subsequent hunk is the meaningful change.

Reply:
Missed that completely, not highlighted well by the GitHub review page.  TYVM!